Gems/AtomTressFX/Code/Passes/HairParentPass.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <Atom/RPI.Public/Pass/ParentPass.h>
+#include <AtomCore/std/containers/array_view.h>
+#include <AzCore/std/containers/fixed_vector.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        //! HairParentPass owns and manages ShadowmapPasses.
```

Comment:
"HairParentPass owns and manages ShadowmapPasses"? Doesn't it own hair passes?

Reply:
:) absolutely, bad copy paste without further comments as was intended.  Added comments and explained why it exists in the first place.